tests/test_regrid.py

```diff
@@ -235,16 +214,14 @@ def test_conservative_nan_thresholds_against_xesmf():
         resolution_lon=2,
     )
     target_dataset = xarray_regrid.create_regridding_dataset(new_grid)
-    regridder = xe.Regridder(ds, target_dataset, "conservative")
+    regridder = xesmf.Regridder(ds, target_dataset, "conservative")
 
     for nan_threshold in [0.0, 0.25, 0.5, 0.75, 1.0]:
-        data_regrid = ds.copy().regrid.conservative(
+        data_regrid = ds.regrid.conservative(
             target_dataset, skipna=True, nan_threshold=nan_threshold
         )
-        data_esmf = regridder(
-            ds.copy(), keep_attrs=True, na_thres=nan_threshold, skipna=True
-        )
-        assert (data_regrid.isnull() == data_esmf.isnull()).mean().values > 0.995
+        data_esmf = regridder(ds, keep_attrs=True, na_thres=nan_threshold, skipna=True)
+        xr.testing.assert_equal(data_regrid.isnull(), data_esmf.isnull())
```

Comment:
NaN thresholding now matches xesmf exactly with the new padding.

Reply:
Nice!